apps/meteor/client/views/admin/rooms/RoomRow.tsx

```diff
@@ -19,12 +20,13 @@ const roomTypeI18nMap = {
 const getRoomDisplayName = (room: Pick<IRoom, RoomAdminFieldsType>): string | undefined =>
 	room.t === 'd' ? room.usernames?.join(' x ') : roomCoordinator.getRoomName(room.t, room);
 
-const RoomRow = ({ room }: { room: Pick<IRoom, RoomAdminFieldsType> }) => {
+const RoomRow = ({ room }: { room: Pick<IRoom, RoomAdminFieldsType> &{ts ?: Date}}) => {
```

Comment:
add the `ts` prop to RoomAdminFieldsType type

Reply:
done